packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/components/health-monitor-form/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,15 @@
+{
+  "octavia_load_balancer_health_monitor_form_name": "Nom",
+  "octavia_load_balancer_health_monitor_form_name_help": "Doit uniquement contenir des nombres, lettres, underscores, tirets ou points.",
+  "octavia_load_balancer_health_monitor_form_type": "Type",
+  "octavia_load_balancer_health_monitor_form_type_default": "Choisissez un type",
+  "octavia_load_balancer_health_monitor_form_expected_code": "Code attendu",
+  "octavia_load_balancer_health_monitor_form_expected_code_help": "Le code HTTP considéré comme un succès",
+  "octavia_load_balancer_health_monitor_form_max_retires_down_tooltip": "Le nombre d'échecs de connexion autorisés avant de marquer le membre comme erreur. Doit être un nombre compris entre 1 et 10. La valeur par défaut est 3. ",
+  "octavia_load_balancer_health_monitor_form_periodicity": "Intervalle",
+  "octavia_load_balancer_health_monitor_form_periodicity_tooltip": "Intervalle entre 2 tests du Health Monitor. Doit être supérieur ou égal au timeout.",
+  "octavia_load_balancer_health_monitor_form_max_retries_tooltip": "Le nombre d'échecs de connexion autorisés avant de marquer le membre comme inactif. Doit être un nombre compris entre 1 et 10.",
+  "octavia_load_balancer_health_monitor_form_timeout_tooltip": "Le temps après lequel un health check s'arrête. Doit être un nombre supérieur ou égal à 0 et inférieur à la périodicité",
+  "octavia_load_balancer_health_monitor_form_cancel": "Annuler",
+  "octavia_load_balancer_health_monitor_form_submit": "Editer"
```

Comment:
Shouldn't it be "Ajouter" ?

Reply:
Nop I change the way the common components work by giving him a submit label. If no submit label is given, the default label is Editer. But is that case , a submit label is send by parent